### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by balanza**
Update fp-ts, jest, typescript

### CODE REVIEW

**Review by gunzip — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "TODO",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "TODO",
   "repository": "https://github.com/teamdigitale/TODO",
```
Comment (gunzip) [inline package.json]:
```suggestion
  "repository": "https://github.com/pagopa/TODO",
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   "author": "https://teamdigitale.governo.it",
```
Comment (gunzip) [inline package.json]:
```suggestion
  "author": "https://www.pagopa.gov.it",
```

**Comment by balanza** [comment]
thanks @gunzip  for spotting TD's references 👍

**Comment by gunzip** [comment]
merged

**Pull request closed without merging**